string name
float64 cx
float64 cy
float64 cz
float64 dx
float64 dy
float64 dz
---

